release_and_branching.md

```diff
@@ -38,11 +38,6 @@ the following extra steps:
 
 * It contains a dependencies.yaml file which drives OLM dependencies to install all other OpenStack service operators
 * When the bundle is generated it runs a script to *merge* all the ENV variables from each service operators controller-manager. These ENV variables contain important webhook defaults that are needed within the OpenStack operator.
-* A csv-merger is used to merge in the CSV from the dataplane-operator along with all the ENV variables harvested from the above.
```

Comment:
I think this part "along with all the ENV variables harvested from the above." is still valid. The csv-merger merges in the collected ENV variable into the CSV.